README.md

```diff
@@ -692,6 +692,43 @@ The official OCS samples are divided in multiple categories depending on the sce
       </table>
     </td>
   </tr>
+  <tr>
```

Comment:
should Assets be its own category? It seems fairly hidden for by being in the bottom of this table

Reply:
I decided to follow the pattern based on data views, which doesn't have its own category. @derekendres, your thoughts? I wouldn't mind separating it but kind of feel data views should be as well.